crates/bindings-csharp/SpacetimeSharpSATS.sln

```diff
@@ -3,24 +3,26 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
 VisualStudioVersion = 17.0.31903.59
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Codegen", "Codegen\Codegen.csproj", "{1E37FF71-567A-4AC8-947E-117098F01142}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Codegen", "Codegen\Codegen.csproj", "{1E37FF71-567A-4AC8-947E-117098F01142}"

```

Comment:
Do we need to worry about these strings for packaging reasons or are they purely internal?

Reply:
Purely internal. I don't even know why VS decided to regenerate them - I tried reverting, but it regenerated them on the next time to the same strings again, so I'm going to keep it.